gallery/src/PageHeader/HostedVersionsSelect.scss

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ 
+.gallery-hosted-versions-select {
+  display: flex;
+  align-items: center;
+  gap: var(--nx-spacing-2x);
+
+  &__label {
+    font-size: 14px;
```

Comment:
Did this sizing come from a designer?

Reply:
> Did this sizing come from a designer?

It approved by a designer but we changed it here recently:
18aab428bcb0badf4d66b08b695127dd78b9a43e